src/ui/EditTask.svelte

```diff
@@ -393,7 +406,7 @@
 Availability of access keys:
 - A: Start
 - B: Before this
-- C:
+- C: Reminder
```

Comment:
I am about to use `C` on `main` for `Created Date`, so this will need to be changed...